Smartway.UiComponent/Buttons/TextButton.xaml

```diff
@@ -1,26 +1,29 @@
 ﻿<?xml version="1.0" encoding="UTF-8"?>
-<Button xmlns="http://xamarin.com/schemas/2014/forms" 
-        xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
-        xmlns:d="http://xamarin.com/schemas/2014/forms/design"
-        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-        xmlns:resources="clr-namespace:Smartway.UiComponent.Resources;assembly=Smartway.UiComponent"
-        mc:Ignorable="d"
-        x:Class="Smartway.UiComponent.Buttons.TextButton"
-        x:Name="Self"
-        HorizontalOptions="Center"
-        VerticalOptions="Center"
-        BackgroundColor="Transparent"
-        TextTransform="None"
-        Font="OpenSansSemiBold"
-        FontSize="15"
-        TextColor="{StaticResource ZgDarkBlue}"
-        HeightRequest="50"
-        Padding="0,15">
-    <Button.Resources>
+<layouts:ClickableItem xmlns="http://xamarin.com/schemas/2014/forms" 
+                       xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+                       xmlns:d="http://xamarin.com/schemas/2014/forms/design"
+                       xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+                       xmlns:layouts="clr-namespace:Smartway.UiComponent.Layouts;assembly=Smartway.UiComponent"
+                       xmlns:resources="clr-namespace:Smartway.UiComponent.Resources;assembly=Smartway.UiComponent"
+                       mc:Ignorable="d"
+                       x:Class="Smartway.UiComponent.Buttons.TextButton"
+                       x:Name="TxtButton"
+                       HeightRequest="50"
+                       TapCommand="{Binding Command, Source={x:Reference TxtButton}}">
+    <layouts:ClickableItem.Resources>
         <ResourceDictionary>
             <ResourceDictionary.MergedDictionaries>
                 <resources:Colors />
             </ResourceDictionary.MergedDictionaries>
         </ResourceDictionary>
-    </Button.Resources>
-</Button>
\ No newline at end of file
+    </layouts:ClickableItem.Resources>
+    <layouts:ClickableItem.Content>
+        <Label Text="{Binding Text, Source={x:Reference TxtButton}}"
```

Comment:
`TxtButton` => `TextButton`

Reply:
si je met "TextButton" ça m'affiche une erreur "Field TextButton is already declared", je pense qu'il le confond avec le nom de la callse